### ISSUE TITLE
[Annotations] Colors from the "Color Picker" panel cannot be selected with keyboard navigation

### ISSUE THREAD

**Issue by SoftVision-CosminMuntean** [mozilla-services/screenshots#4164]
[Affected versions]:
- Nightly 60.0a1
- Screenshots 31.0.0 stage

[Affected Platforms]:
- All Windows
- All Mac
- All Linux

[Prerequisites]:
- The "xpinstall.signatures.required" boolean pref is set to "false".
- The "extensions.legacy.enabled" boolean pref is set tot "true".
- Have the Screenshots stage version installed on a new clean profile.
- Have a saved shot link.

[Steps to reproduce]:
1. Open Firefox browser with the profile from prerequisites and navigate to the saved shot link.
2. Click the "Edit This Image" button.
3. Press "Tab" key 4 times in order to focus the "Color Picker" button and press "Enter".
4. Press the "Tab" key and observe the behavior.

[Expected result]:
- The first color form the "Color Picker" panel gets focused.

[Actual results]:
- The "Clear" button gets focused.

[Notes]:
- Here is a screen recording with the issue:
![color picker keyboard navigation](https://user-images.githubusercontent.com/19226534/36843362-63422b1c-1d57-11e8-90df-0f9957969350.gif)

**Comment by wresuolc** [comment]
After choosing a color you'd still need to apply it to the image with the mouse, so we're considering this a wontfix.
